tests/traced_data/resources/coda_export_expected_output_coded.csv

```diff
@@ -0,0 +1,5 @@
+id;owner;data;timestamp;schemeId;schemeName;deco_codeValue;deco_codeId;deco_confidence;deco_manual;deco_timestamp;deco_author
+0;0;female;;;;;;;;;
+1;1;m;;;;;;;;;
+2;2;WoMaN;;;;;;;;;
+3;3;27;;;;;;;;;
```

Comment:
I don't remember what we discussed about this - could we ensure these all have a trailing new line at the end of the lines?

Reply:
Not without breaking Coda. There is an open issue in Coda for this (https://github.com/AfricasVoices/coda/issues/137), but I didn't think fixing it was an immediate priority.